tools/BuildTools.proj

```diff
@@ -0,0 +1,17 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(MSBuildThisFileDirectory)\PushToBlobFeed.targets" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp1.0</TargetFramework>
+    <CopyBuildOutputToOutputDirectory>false</CopyBuildOutputToOutputDirectory>
+    <CopyOutputSymbolsToOutputDirectory>false</CopyOutputSymbolsToOutputDirectory>
+    <GenerateDependencyFile>false</GenerateDependencyFile>
+    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
+    <NonShipping>true</NonShipping>
+    <NoStdLib>true</NoStdLib>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.Build.Tasks.Feed" Version="1.0.0-prerelease-02203-01" />
```

Comment:
One suggestion is to move this PackageReference into "PushToBlobFeed.targets" so that this entry build tools proj  just contains some general properties and a single import per tool project (assuming there may be more at some point).  That would be reducing the connections here and make the feed stuff event more contained.  I'm not fully convinced this is a better way to go though since leaving the packagereference here may make things simpler in the future for updates if there are multiple packagereferences (and correspondingly multiple imports).  I'm wavering on whether there's actually value in moving this packagereference.  I'll leave it up to you.